docs/Creating-Your-Own-EmulationStation-Theme.md

```diff
@@ -1,166 +1,153 @@
 
-In this tutorial I'm going to take you through the basics of creating a theme for Emulationstation and RetroPie.
+During the course, three systems will be covered as example:
 
-We are going to create a simple theme called 'Spare'. We'll be using the default 'Carbon' theme as a guide, but creating all the XML from scratch so we can get a better understanding of it.
+- Game Boy (folder: `gb`)
+- Nintendo Entertainment System (folder: `nes`)
+- Super Nintendo Entertainment System (folder: `snes`)
 
-We'll be creating folders for 3 systems:
+These systems are not only ubiquitous but contain some graphically layout challenges, for example the different shaped box art (square, portrait and landscape respectively), and we will be using those differences for some more advanced concepts in a later tutorial.
 
-- Gameboy (gb)
-- Nintendo Entertainment System (nes)
-- Super Nintendo Entertainment System (snes)
+After the exercise with three theme-layouts for these systems you should have got an understanding how theming works in general. Three is the minimum of systems to fill the carousel and also the maximum visible at the same time.
 
-I chose these 3 as they are pretty ubiquitous in the console world and most people should have heard of them. I also chose these 3 as they all have different shaped box art (square, portrait and landscape respectively), and we will be using those differences for some more advanced concepts in a later tutorial.
+The theme you are going to make will look like this:
 
-We are only doing 3 folders because once you've done one, the rest are pretty similar, and this tutorial is pretty long as it is, and you can only see 3 systems on the carousel anyway.
+<figure markdown>
+  ![Spare Theme System View](https://i.imgur.com/72DcwpD.jpg)
+  <figcaption>Spare Theme System View</figcaption>
+</figure>
 
-The theme we are going to make will look like this:
+<figure markdown>
+  ![Spare Theme Basic View](https://i.imgur.com/lSIjOyk.jpg)
+  <figcaption>Spare Theme Basic View</figcaption>
+</figure>
 
-![Spare Theme System View](http://i.imgur.com/72DcwpD.jpg)
-*['Spare' Theme System View]*
-
-![Spare Theme Basic View](http://i.imgur.com/lSIjOyk.jpg)
-*['Spare' Theme Basic View]*
-
-![Spare Theme Detailed View](http://i.imgur.com/r5SHwkc.jpg)
-*['Spare' Theme Detailed View]*
+<figure markdown>
+  ![Spare Theme Detailed View](https://i.imgur.com/r5SHwkc.jpg)
+  <figcaption>Spare Theme Detailed View</figcaption>
+</figure>
 
 It is a simple, clean theme, not very outlandish, but it's enough to teach the essentials, and a few of the more advanced ideas.
 
----
+**Note**
 
-## Notes
+> This tutorial was developed on Windows, but all the main parts (the XML and images) can be done on any Operating System, as long as you can put or assess the files into the `theme` folder on your Raspberry Pi (e.g., with Samba, see [setup](#use-samba-share-of-emulationstations-themes-on-retropie)).
```

Comment:
Not sure what _put or assess_ means, but I'm thinking you meant something along the lines of _put your assets_ ?

Reply:
Rephrased it, was thinking of "accessing".